tauri-app/package.json

```diff
@@ -1,26 +1,46 @@
 {
-  "name": "tauri-app",
-  "private": true,
-  "version": "0.0.0",
-  "type": "module",
-  "scripts": {
-    "dev": "vite",
-    "build": "vite build",
-    "preview": "vite preview",
-    "check": "svelte-check --tsconfig ./tsconfig.json",
-    "tauri": "tauri"
-  },
-  "dependencies": {
-    "@tauri-apps/api": "^1.5.2"
-  },
-  "devDependencies": {
-    "@sveltejs/vite-plugin-svelte": "^3.0.1",
-    "@tsconfig/svelte": "^5.0.2",
-    "svelte": "^4.2.8",
-    "svelte-check": "^3.4.6",
-    "tslib": "^2.6.0",
-    "typescript": "^5.0.2",
-    "vite": "^5.0.0",
-    "@tauri-apps/cli": "^1.5.8"
-  }
+	"name": "tauri-app",
+	"version": "0.0.1",
+	"private": true,
+	"scripts": {
```

Comment:
why npm scripts?

Reply:
generated by scaffolding and kept for reference but yes its redundant and unnecessary. removed.

Reply:
reverted -- added npm scripts back in to ensure users are not running incorrect versions of node dependency commands by using `npx`